modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -19,19 +19,27 @@ Cluster Logging Operator and Elasticsearch Operator.
 
 .Prerequisites
 
-Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
-requires its own storage volume.
+You are provided with a maximum of 600GiB of persistent storage exclusively
+available for cluster logging related Elasticsearch deployments. Note that each
+Elasticsearch node requires its own storage volume.
 
-Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
-The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
-{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
-memory setting though this is not recommended for production deployments.
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs
+8G of memory for both memory requests and limits. Each Elasticsearch node can
+operate with a lower memory setting though this is not recommended for
+production deployments.
 
 .Procedure
 
-. Install the Elasticsearch Operator via the OperatorHub. See: xref:../operators/olm-adding-operators-to-cluster.adoc[Adding Operators to a cluster].
+. Install the Elasticsearch Operator via the OperatorHub:
```

Comment:
s/via/from

> ! via prep
Use this term, which means “by way of,” to refer to routing data, such as in “Messages are routed via
SMTP.” In other situations, try to replace “via” with one of the following synonyms: “across,” along,” “by,”
“from,” “on,” or “through.”

Reply:
Agreed. Done!